enterprise/backend/test/metabase_enterprise/sandbox/models/params/field_values_test.clj

```diff
@@ -54,18 +56,16 @@
           (testing "after changing the question, should create new FieldValues"
             (let [new-query (mt/mbql-query categories
                                            {:filter [:and [:> $id 1] [:< $id 4]]})]
-              ;; sleeping should ensure that updated_at changes
-              (Thread/sleep 1)
-              (t2/update! Card card-id {:dataset_query new-query}))
+               (t2/update! Card card-id {:dataset_query new-query
+                                         :updated_at    (t/local-date-time)}))
```

Comment:
> How does this make sure the timestamp changes?
the previous sleep for 1ms so I don't think it makes a difference in having it right? 


> Wouldn't the default update do the same?

Unfortunately, it's not because the whole test is under a transaction, in real production we don't need to update the `updated_at` tho, but we need it in the test.

check https://github.com/metabase/metabase/pull/33738#discussion_r1324181573 for more info

Reply:
👍 about explicitly setting `:updated_at`.
I still don't see why removing the sleep is OK. It does assume a timestamp of at least millisecond precision.

Reply:
okay I put back the sleep in [6277e34](https://github.com/metabase/metabase/pull/33738/commits/6277e340f2c3df80f26e4fb8da5e2bca13328dd2)